examples/example-02/README.md

```diff
@@ -113,8 +113,9 @@ const keySharesData = {
   },
 };
 
-// Create KeyShares instance using key shares data
-const keyShares = await KeyShares.fromData(keySharesData);
+// Create SSVKeys instance using key shares data
+const ssvKeys = new SSVKeys(SSVKeys.VERSION.V2);
+const keyShares = ssvKeys.keySharesInstance.init(keySharesData);
```

Comment:
`ssvKeys.keySharesInstance.init` - is more developer specific type of thinking.
But the library will be used by newbies, people who come from universities, PHDs etc, who might be not that type of mind.

Let's do interface more user-friendly then. Just as an example:

```
class SSVKeys {
    get keyShares() {
        if (!this.keyShares) this.keyShares = new KeyShares(this);
    }
}

class KeyShares {
    private ssvKeys: ISSVKeys;
    
    constructor(ssvKeys: ISSVKeys) {
        // to have access to encryption methods etc
        this.ssvKeys = ssvKeys;
    }

    fromJson(json: string | object) { ... }
    toJson() {...}
    validate() {... validating everything by calling each specific validation method ... }
    build() { ... building using this.ssvKeys methods, in case if it contains all necessary data ... }
}
```

Then final line will be more intuitive and less specific:

`const keyShares = ssvKeys.keyShares.fromJson(keySharesData);`

Also not required but would be great to do:

- `class SharesEncryption` - for all encryption methods + validation. Encryption will use all the threshold and other classes. Validation should/can be extracted to helpers or `SharesValidation` class
- `class KeyShares` - for keyshares file
- `class SSVKeys` - orchestrating all upper classes without knowledge of any specifics